dlt/cli/plugins.py

```diff
@@ -645,6 +647,29 @@ def execute(self, args: argparse.Namespace) -> None:
             fmt.note("Docs page updated.")
 
 
+class AiCommand(SupportsCliCommand):
+    command = "ai"
+    help_string = "Use AI-powered development tools and utilities"
+    # docs_url = 
+    description = "The `dlt ai` command provides commands to launch the MCP server and "
+
+    def configure_parser(self, ai_cmd: argparse.ArgumentParser) -> None:
+        self.parser = ai_cmd
+
+        ai_cmd.add_argument("ide", choices=SUPPORTED_IDES)
+
+        ai_subparsers = ai_cmd.add_subparsers(
+            title="Available subcommands", dest="operation", required=False
+        )
+        ai_subparsers.add_parser(
+            "setup", help="Generate IDE-specific configuration and rules files",
+        )
+        # TODO support MCP-proxy configuration
+        # ai_mcp_cmd = ai_subparsers.add_parser("mcp", help="Launch the dlt MCP server")
+    
+    def execute(self, args: argparse.Namespace) -> None:
+        ai_setup_command_wrapper(args.ide)
```

Comment:
should we check if project is initialized already? ie. if `.dlt` folder exists (you check if run_context.settings exist)